src/eda_server/api/activation.py

```diff
@@ -26,9 +27,30 @@
 router = APIRouter(tags=["activations"])
 
 
+def activation_unprocessable_entity_exception(e):
+    error_detail = e.orig.args[0].split("\n")[1]
+    object_id = re.findall(r"\d+", error_detail)[0]
+    dependent_objects = ["rulebook", "inventory", "extra_var", "project"]
+    for object in dependent_objects:
+        if object in error_detail:
+            return JSONResponse(
```

Comment:
Should it return a generic exception if dependent object is not found?

Reply:
We are using this approach when we hit 500 docker error on activation creation instance, so I was trying to keep it consistent with that one.